ios/climbARmobile.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What's this for?

Reply:
this is something that xcode spits out after running a build, will probably need to delete/ .gitignore it